docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -660,206 +763,93 @@
-    "| :------------------------ | :---------- | :------------------------------------------------------------------------------------------------------------------- | :------- | :------ |\n",
-    "| n_recover_iteration       | int         | The number of recovery iterations                                                                                   | No       | 5       |\n",
-    "| n_batch                   | int         | The number of batches generated per iteration                                                                       | No       | 1       |\n",
-    "| number_of_states_pick_out | int or None | The dimension of the QSCI vector in the recovery process. If None, it is set to the same as the value used for QSCI | No       | None    |\n",
-    "| recovery_r_multiple       | int         | The multiplier for number_of_states_pick_out. This is the number of states selected from the raw sample             | No       | 10      |\n",
-    "\n",
-    "To activate configuration recovery, specify `configuration_recovery = True` in the error mitigation field, or customize any of the above fields. For example:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "f78bd298-35de-4513-baa8-a0c5b3706815",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Use default configuration recovery setting\n",
-    "default_mitigation_setting = {\n",
-    "    \"mitigation_setting\": {\n",
-    "        \"configuration_recovery\": True\n",
-    "    }\n",
-    "}\n",
-    "\n",
-    "# Customized configuration recovery setting\n",
-    "customized_mitigation_setting = {\n",
-    "    \"mitigation_setting\": {\n",
-    "        \"configuration_recovery\": {\n",
-    "            \"n_recover_iteration\": 20,\n",
-    "            \"n_batch\": 10,\n",
-    "            \"number_of_states_pick_out\": 5000\n",
-    "        }\n",
-    "    }\n",
-    "}"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "b09639e2-3d04-4b1d-be98-ae17432c1b4c",
-   "metadata": {},
-   "source": [
-    "<Admonition type=\"note\">You cannot enable any other mitigation methods when `configuration_recovery` is turned on.</Admonition>"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "9f9e75aa-cb2d-4225-96cf-343db6de6781",
    "metadata": {},
    "source": [
     "## Performance\n",
     "\n",
-    "### $\\text{N}_2$ dissociation curve\n",
+    "### N2 dissociation curve\n",
     "\n",
-    "As a performance benchmark, we provide the 20-qubit $\\text{N}_2$ dissociation curve as a demonstration.\n",
+    "As a performance benchmark, we provide the 20-qubit $\\text{N}_2$ dissociation curve as a demonstration. \n",
     "\n",
-    "![N2_dissociation](/images/guides/qunasys-qsci/N2_dissociation.svg)\n",
+    "![N2_dissociation](/images/guides/qunasys-qsci/N2_dissociation.png)\n",
```

Comment:
Instead of "N2_dissoviation", this should describe what the image shows.  Also, the svg images should always be used when possible.